### DISCUSSION TITLE
Welcome to Power Tab Editor Discussions!

### DISCUSSION THREAD

**Discussion by cameronwhite** [powertab/powertabeditor#317] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by Canaanabolaanan** [comment]
Ah, excellent. Thank you for the quick email response. I haven't had a chance to check it out to see if it works yet, but I shall. In the meantime, I figure this might be a good opportunity to ask the community if anyone has any interest in making a Chrome OS version. I don't know if it's more work than it's worth, but it probably doesn't hurt to at least float the idea. Thanks and cheers!

**Comment by AxxlForce** [comment]
Excellent project, @devs thank you very much! I have a couple of questions:

1. How can I select bars over multiple rows
2. How can I add a new bar and how can I paste multiple bars after the current bar

I hope these questions haven't been addressed yet at least I didn't find it.

**Reply by cameronwhite** [reply]
Thanks!
I think selecting bars across multiple systems is bug #46, so currently you'd have to copy per-staff.
You can add new bars by pressing B (or Shift+B to open the dialog for other barline types), and can use the options in the Position menu if you need to shift notes around to make room

**Reply by AxxlForce** [reply]
Inserting a bar works, but then you get more bars per row and they are not reformatted when using "polish score" and when there is too many bars in one row this stops working at some point. The main problem is that I don't now how to make space/insert notes after the last note in the score and/or how to move a bar to the next row.

**Reply by cameronwhite** [reply]
Could you file some bugs for those? (especially where "polish system" doesn't seem to work)

**Comment by Ebdim9th** [comment]
this is kinda what i needed, I find myself asking several questions at once, and not all of them related. i think i would like to see a regular splash cymbal added as opposed to just the backwards one, just enough to round out the synth drums, especially when there are parts where i'd want the metronome cymbal, which usally substitutes, to fall silent. No bending over backwards and sideways to add a drumkit, just that one extra drop down choice...

**Comment by Ebdim9th** [comment]
I /do/ hate being a pain, which is why I haven't said anything, but the little panning knobs still arent working for me, love using the dark skin though, looks better to me....

**Reply by cameronwhite** [reply]
Does https://github.com/powertab/powertabeditor/actions/runs/584827055 work?

**Comment by Ebdim9th** [comment]
im not sure, i see one of for mac and one for windows and a couple of others but if it was updated around two weeks ago thats what i downloaded, i tried x86 instead of x64 just now, it didn't make any difference....

**Reply by cameronwhite** [reply]
Could you let me know what the full version number in the title bar is?

**Reply by Ebdim9th** [reply]
it's v2.2.0-alpha13-28-ga24fd344

the folder says 14 but that says 13

or maybe it doesn't......

ah the downloads page does at the top....

**Reply by cameronwhite** [reply]
That sounds like https://github.com/powertab/powertabeditor/commit/a24fd344605b77a203192d72bdd1957c16496676 which is the actual alpha 14 release, so it doesn't sound like you're using the latest development build that would have the fixes for the pan knobs.

**Reply by Ebdim9th** [reply]
Downloaded it, still no change. Very odd, Alpha 8 on my XP desktop had the adjusting pan knobs working.....

**Reply by cameronwhite** [reply]
Is the version number still the same as before? Maybe try doing a full uninstall first?

**Reply by Ebdim9th** [reply]
a full uninstall.... easy first solution..... I'll try it when I get home....

**Reply by cameronwhite** [reply]
I just meant removing any existing installations of Power Tab Editor, which should be pretty quick. I'm wondering if the installer isn't properly replacing old versions.

**Reply by Ebdim9th** [reply]
i uninstalled both an older version and that one, then reinstalled it, but no difference....

**Reply by cameronwhite** [reply]
Hmm. To confirm, you're installing the latest build from https://github.com/powertab/powertabeditor/actions, not from https://github.com/powertab/powertabeditor/releases? The pan fixes have been made since the last release (alpha 14)

**Reply by Ebdim9th** [reply]
Just tried it from that page ( /actions ),... it still says alpha 13 at the top.... I just don't know.....

**Reply by Ebdim9th** [reply]
x64 Hah! It works!! Victory ... thank you muchly Cameron, sorry for the confusion.... ETA and it now reads Alpha 14 too....

**Comment by DrQuest** [comment]
Having trouble installing. Is there a guide somewhere?

**Reply by cameronwhite** [reply]
There isn't anything specific, but this is a good place to ask for help!
Which platform are you on (and which installer did you download?)

**Reply by DrQuest** [reply]
Thanks. I tried both the 64 and 86 versions of Alpha 14 and both failed on install.

**Reply by DrQuest** [reply]
![image](https://user-images.githubusercontent.com/56480379/113455051-85099880-93be-11eb-94c6-089990b59673.png)

**Reply by cameronwhite** [reply]
Hmm, that sounds like the issue from #305 that should have been fixed in v14. There's a workaround in that thread of manually installing the vcredist exe

**Reply by DrQuest** [reply]
I get the same fault when I try to install 11 through 14..

**Reply by cameronwhite** [reply]
Does it work after installing vcredist from https://support.microsoft.com/en-us/topic/the-latest-supported-visual-c-downloads-2647da03-1eea-4433-9aff-95f26a218cc0 ?

**Reply by DrQuest** [reply]
That fixed it for 12. Going to try 14 now. Thanks

**Comment by psyomn** [comment]
This would add some extra managerial overhead, but how would you feel about a discord server? I wonder if there's a way we could do something like that with extra low commitment.

**Reply by cameronwhite** [reply]
Sure, I don't mind that if there's some interest in it

**Reply by Ebdim9th** [reply]
I'm in, I'd like to see what everybody's doing with PowerTab, posted either here or on a Discord channel....

**Reply by cameronwhite** [reply]
I've just set one up - https://discord.gg/mpXfquuzxe
Let me know if the link works, I haven't used discord too much :)

**Reply by psyomn** [reply]
It works! I'm on there right now!

**Reply by Ebdim9th** [reply]
im there too!

**Comment by Ebdim9th** [comment]
ok, in the latest version, 14-58, the bends don't work. they don't show up or play back.

**Reply by Ebdim9th** [reply]
working with 14-41 how do you get only a gradual bend, no release? the latter is easy to get the former, not so much....

**Reply by cameronwhite** [reply]
Could you try https://github.com/powertab/powertabeditor/actions/runs/758720662? I think the issue might have been introduced in the very latest commit which changed some things in the bend dialog.

**Reply by cameronwhite** [reply]
Should hopefully be fixed now with https://github.com/powertab/powertabeditor/actions/runs/867316969

**Reply by Ebdim9th** [reply]
checking it out now.....

**Reply by Ebdim9th** [reply]
now the playback is really low in volume....

**Comment by CrushedLeaves** [comment]
this application is very good. but I can't use it. I wish developer writer a usebook.

**Reply by cameronwhite** [reply]
I'm not quite sure what this means: `writer a usebook`
Could you please explain a bit more?

**Comment by beaudoiin** [comment]
Hello Cameron, are you still actively maintaining this app?
Thank you so much for your hard work!

**Reply by cameronwhite** [reply]
Yes it's still being maintained, although the activity level varies depending on my free time 😀 
New contributors are also always welcome!

**Comment by beaudoiin** [comment]
Well I just want to say thank you. I think ypur work is very cool and im
glad some one remade the program, if I have any ideas I'll share them. I'm
just learning how to program Java, no where near to C. But either way good
job!

**Comment by nshobe** [comment]
Just finding this project after many decades now of using the ol' 1.7 PT. Is there a way to view scores with multiple pages side by side? Back in the day we were all using 4:3 monitors, and infinite scroll made sense, but in the modern 16:9 era it would be nice to see two pages at a time, side by side.

**Reply by cameronwhite** [reply]
It's still a linear layout currently, although you do have some control over zooming
